packages/smooth_app/lib/cards/product_cards/product_title_card.dart

```diff
@@ -49,17 +49,18 @@ class ProductTitleCard extends StatelessWidget {
     return Align(
       alignment: Alignment.topLeft,
       child: InkWell(
-        onTap: () async {
-          if (getProductName(product, appLocalizations) ==
-              appLocalizations.unknownProductName) {
-            await Navigator.push<bool>(
-              context,
-              MaterialPageRoute<bool>(
-                builder: (BuildContext context) => AddBasicDetailsPage(product),
-              ),
-            );
-          }
-        },
+        onTap: (getProductName(product, appLocalizations) ==
```

Comment:
For the record, `ListTile`s have an `onPressed` field: we don't need the `InkWell`.